system/loggerd/tests/test_loggerd.py

```diff
@@ -284,10 +284,10 @@ def test_userflag(self):
 
     # sleep enough for the first poll to time out
     # TODO: fix loggerd bug dropping the msgs from the first poll
-    # managed_processes["loggerd"].start()
-    # for s in services:
-    #   while not pm.all_readers_updated(s):
-    #     time.sleep(0.1)
+    managed_processes["loggerd"].start()
+    for s in services:
+      while not pm.all_readers_updated(s):
```

Comment:
I wrote a helper for this today https://github.com/commaai/cereal/commit/cf9c5cbb9196f80a25bd8421a9921ce4801a7561